src/analytics/lambdas/load-data-workflow/load-manifest-to-redshift.ts

```diff
@@ -100,7 +99,7 @@ export const handler = async (event: LoadManifestEvent, context: Context) => {
     process.env.REDSHIFT_CLUSTER_IDENTIFIER!,
   );
 
-  for (var i=0;i < jobList.entries.length; i++) {
+  for (let i=0;i < jobList.entries.length; i++) {
```

Comment:
change to `for (const entriy of jobList.entries)` is better

Reply:
done